crosswalk/index.html

```diff
@@ -856,5 +878,9 @@ <h2>Schema.org and ONIX</h2>
 				</tbody>
 			</table>
 		</section>
+        <section>
+            <h2>Acknowledgements</h2>
+            <p>Christopher Saynor (EDItEUR)</p>
```

Comment:
If you're only acknowledging one person, you might want to turn this into a paragraph instead. Maybe something like: "The editors would like to thank Christopher Saynor (Editeur) for his invaluable help reviewing this document."